EIPS/eip-5164.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 5164
+title: Cross-Chain Execution
+description: This specification defines an interface that supports execution across EVM networks.
+author: Brendan Asselstine (@asselstine), Anna Carroll (@anna-carroll), Hadrien Croubois (@Amxx), Nam Chu Hoai (@nambrot), Georgios (@geogons), Theo Gonella (@mintcloud), Rafael Solari (@rsolari), Auryn Macmillan (@auryn-macmillan), Nathan Ginnever (@nginnever)
```

Comment:
@Pandapip1 @SamWilsn is it technically feasible to exclude author line from spell checking, I wonder?

Reply:
Technically... not as of right now. CodeSpell has been temporarily bypassed though, so I guess this is good to be merged.